### PR TITLE
wandlowski_1996_structural_10277

### PR DESCRIPTION
**Pull request by markus-chem**
add lit folder

### CODE REVIEW

File: literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277.bib [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@article{wandlowski_1996_structural_10277,
+  title={Structural and kinetic aspects of bromide adsorption on Au (100)},
```
Comment (DunklesArchipel) [inline literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277.bib:2]:
```suggestion
  title={Structural and kinetic aspects of bromide adsorption on Au(100)},
```

File: literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_dashed.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    
+curator:
+    version: 1
+    name: Markus Schilling
+    orcid: https://orcid.org/0000-0003-2684-3711
+    digitized: 2021-11-04 # YYYY-MM-DD
+    
+figure description: #This section will be updated by module `CV`
+    version: 1
+    linked measurements: in-situ surface X-ray # FTIR, SXRD, Ring Disc - indicate if any other operando methods were shown in the same plot, which is linked to the data
```
Comment (DunklesArchipel) [inline literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_dashed.yaml:20]: "[in situ](https://en.wikipedia.org/wiki/In_situ)" is usually written without a dash. (No dash in this latin wording)

Comment (DunklesArchipel) [reply literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_dashed.yaml:20]: Is surface X-Ray abbreviated as SXRD in the manuscript?

File: literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_solid.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    
+curator:
+    version: 1
+    name: Markus Schilling
+    orcid: https://orcid.org/0000-0003-2684-3711
+    digitized: 2021-11-04 # YYYY-MM-DD
+    
+figure description: #This section will be updated by module `CV`
+    version: 1
+    linked measurements: in-situ surface X-ray # FTIR, SXRD, Ring Disc - indicate if any other operando methods were shown in the same plot, which is linked to the data
```
Comment (DunklesArchipel) [inline literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_solid.yaml:20]: see above

**Review by DunklesArchipel — APPROVED** [review]
Aside from a few comments on "in situ" and "SXRD" this is fine.

File: literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_dashed.yaml (hunk 2 above)
Comment (markus-chem) [reply literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_dashed.yaml:20]: The description was taken from the first sentence of the abstract of that paper (with dash). I am fine with both.
SXRD is single crystal XRD for me, which is different in that case. SXRD is not used in the paper.

How should I change it?

File: literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_solid.yaml (hunk 3 above)
Comment (markus-chem) [reply literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_solid.yaml:20]: see above

File: literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_dashed.yaml (hunk 2 above)
Comment (DunklesArchipel) [reply literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_dashed.yaml:20]: So it seems that the authors write "in situ surface x-ray diffraction" in the introduction and "surface x-ray scattering - SXS" in the manuscript. So probably you should simply change it to SXRD, which is commonly used nowadays.

File: literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_solid.yaml (hunk 3 above)
Comment (DunklesArchipel) [reply literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_solid.yaml:20]: SXRD, see above

File: literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_dashed.yaml (hunk 2 above)
Comment (markus-chem) [reply literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_dashed.yaml]: changed to SXRD

File: literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_solid.yaml (hunk 3 above)
Comment (markus-chem) [reply literature/wandlowski_1996_structural_10277/wandlowski_1996_structural_10277_p1_1a_solid.yaml]: changed to SXRD

**Pull request merged**